### PR TITLE
Show more columns on partners page on mobile

### PR DESCRIPTION
**Pull request by hacodeorg**
https://codedotorg.atlassian.net/browse/PLC-365

Preview:
Resize partners page

### CODE REVIEW

**Comment by agealy** [comment]
Nice gif 👍

File: pegasus/sites.v3/code.org/public/css/logos.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* 970px and down (tablet and phones): show 3 columns */
+
+@media screen and (min-width: 0px) and (max-width: 970px)
+{
+    .col-mobile-30 { width: 30%; float:left; }
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/css/logos.css]: nit: two spaces per tab is usual.

File: pegasus/sites.v3/code.org/public/css/logos.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/* 970px and down (tablet and phones): show 3 columns */
+
+@media screen and (min-width: 0px) and (max-width: 970px)
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/css/logos.css]: Isn't the range for this technically 513px to 970x since 512px and below will be dealt with below?

Comment (hacodeorg) [reply pegasus/sites.v3/code.org/public/css/logos.css]: Yes. I'm following the comment style in `brstrap.css` here. What is a better way to communicate it, by updating comment or updating `min-width`?

Comment (breville) [reply pegasus/sites.v3/code.org/public/css/logos.css]: Oh this is referring to the actual range you specify in the `@media` query.  If you know you're going to be immediately overridden, you can avoid that by specifying correct `min-width` values.  That helps avoids seeing lots of CSS overriding other CSS when debugging.  Not a big deal, but something to keep in mind.

**Review by breville — APPROVED** [review]

File: pegasus/sites.v3/code.org/public/css/logos.css (hunk 2 above)
Comment (hacodeorg) [reply pegasus/sites.v3/code.org/public/css/logos.css]: Got it. I'll make it clearer.

**Pull request merged**